tests/manage/z_cluster/test_change_policy_scale_up_down.py

```diff
@@ -0,0 +1,142 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.utility.utils import TimeoutSampler, run_cmd
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.helpers import helpers
+from ocs_ci.helpers.helpers import wait_for_resource_state
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.framework.testlib import ManageTest, tier2, bugzilla
+from ocs_ci.utility import templating
+from ocs_ci.ocs.resources.pod import get_pod_obj
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2024870")
+class TestChangePolicyScaleUpDown(ManageTest):
+    """
+    Test Change Policy Scale Up/Down
+    """
+
+    @pytest.fixture(scope="function", autouse=True)
+    def teardown(self, request):
+        def finalizer():
+            self.delete_pod(pod_name="simple-app", namespace=self.project_name)
+
+        request.addfinalizer(finalizer)
+
+    def test_change_policy_scale_up_down(
+        self,
+        teardown_project_factory,
+        pvc_factory,
+    ):
+        self.project_name = "test-project"
+        self.ocp_obj = ocp.OCP(namespace=self.project_name)
+        project_obj = helpers.create_project(project_name=self.project_name)
+        teardown_project_factory(project_obj)
+
+        self.pvc_obj = pvc_factory(
```

Comment:
We have dc_pod_factory for deployment config related pods, do you think it is a good idea to pull out the `pod_dict_path` arg into the factory args so that you can simply call the dc_pod_factory with a change in pod_dict_path?
This way you can remove the lines of code from 36 to 72. It can also help others if they wanted to deploy pods with new deployment configs